backend/routes/cms/getArticleJSON.js

```diff
@@ -0,0 +1,24 @@
+const Article = require('../../models/article');
+
+async function getArticleJSON(req, res) {
+  // route /api/v1/articles/:id
+  const { id } = req.params;
+  let query = {};
+
+  // valid or not Mongo ObjectID
+  if (id && id.match(/^[0-9a-fA-F]{24}$/)) {
+    query = { _id: id };
+  }
+
```

Comment:
I think we should respond with failure if the id is supplied not valid. Not a good idea to just return all users me things

Reply:
users?

Reply:
looked deeper if you mean all articles is bad,
request with 'id' is not in use by app. (frontend/src/actions.js line 108 is only place which requests from server)
that block of code will never run. can be safely deleted while refactoring stage.
I think that was for development if i start server and manually type request into address bar, to see that it returns json.